src/patternfly/components/Content/styles.scss

```diff
@@ -40,70 +40,70 @@
   h5,
   h6 {
     margin: 0;
-    font-weight: $pf-global--normal--FontWeight;
-    line-height: $pf-global--md--LineHeight;
+    font-weight: $global--FontWeight--normal;
+    line-height: $global--LineHeight;
   }
 
   h1 {
-    margin-bottom: $pf-global--spacer--xs;
-    font-size: $pf-global--hero-title--FontSize;
+    margin-bottom: $global--spacer;
+    font-size: $global--FontSize--xxxl;
     &:not(:first-child) {
-      margin-top: $pf-global--spacer--lg;
+      margin-top: $global--spacer--xl;
     }
   }
 
   h2 {
-    margin-bottom: $pf-global--spacer--xs;
-    font-size: $pf-global--main-title--FontSize;
+    margin-bottom: $global--spacer;
+    font-size: $global--FontSize--xxl;
     &:not(:first-child) {
-      margin-top: $pf-global--spacer--xl;
+      margin-top: $global--spacer--xxl;
     }
   }
 
   h3 {
-    margin-bottom: $pf-global--spacer--xs;
-    font-size: $pf-global--secondary-title--FontSize;
+    margin-bottom: $global--spacer;
+    font-size: $global--FontSize--xl;
     &:not(:first-child) {
-      margin-top: $pf-global--spacer--lg;
+      margin-top: $global--spacer--xl;
     }
   }
 
   h4 {
-    margin-bottom: $pf-global--spacer--md;
-    font-size: $pf-global--sub-title--FontSize;
+    margin-bottom: $global--spacer--lg;
+    font-size: $global--FontSize--lg;
   }
 
   h5 {
-    margin-bottom: $pf-global--spacer--md;
-    font-size: $pf-global--body--FontSize;
+    margin-bottom: $global--spacer--lg;
+    font-size: $global--FontSize--md;
   }
 
   h6 {
-    margin-bottom: $pf-global--spacer--md;
-    font-size: $pf-global--caption--FontSize;
+    margin-bottom: $global--spacer--lg;
+    font-size: $global--FontSize--sm;
   }
 
   blockquote {
-    padding: $pf-global--spacer--md;
-    border-left: 5px solid $pf-global-dark-off-color;
+    padding: $global--spacer--lg;
+    border-left: 5px solid $global--Color--dark-alt;
```

Comment:
I think this one was probably meant to be something else. It's using a variable that is only meant to be used with the color property.